src/main/java/com/gmail/shablinski93/model/Dish.java

```diff
@@ -1,43 +1,83 @@
 package com.gmail.shablinski93.model;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
-
-import javax.persistence.*;
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 import java.util.UUID;
 
-@Data
-@NoArgsConstructor
-@AllArgsConstructor
-@EqualsAndHashCode
-@Entity
-@Table(name = "dish")
 public class Dish implements Serializable {
 
-    @Id
+    public Dish() {
+    }
+
     private UUID dishId;
     private String dishName;
     private Integer caloriesCount;
     private Integer ingredientCount;
-    @ManyToMany
-    @JoinTable(name = "dish_ingredient",
-            joinColumns = @JoinColumn(name = "mainDishId"),
-            inverseJoinColumns = @JoinColumn(name = "mainIngredientId")
-    )
+
     List<Ingredient> ingredients = new ArrayList<Ingredient>();
 
+    public UUID getDishId() {
+        return dishId;
+    }
+
+    public void setDishId(UUID dishId) {
+        this.dishId = dishId;
+    }
+
+    public String getDishName() {
+        return dishName;
+    }
+
+    public void setDishName(String dishName) {
+        this.dishName = dishName;
+    }
+
+    public Integer getCaloriesCount() {
+        return caloriesCount;
+    }
+
+    public void setCaloriesCount(Integer caloriesCount) {
+        this.caloriesCount = caloriesCount;
+    }
+
+    public Integer getIngredientCount() {
+        return ingredientCount;
+    }
+
+    public void setIngredientCount(Integer ingredientCount) {
+        this.ingredientCount = ingredientCount;
+    }
+
+    public List<Ingredient> getIngredients() {
+        return ingredients;
+    }
+
+    public void setIngredients(List<Ingredient> ingredients) {
+        this.ingredients = ingredients;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Dish dish = (Dish) o;
+        return Objects.equals(dishId, dish.dishId) && Objects.equals(dishName, dish.dishName) && Objects.equals(caloriesCount, dish.caloriesCount) && Objects.equals(ingredientCount, dish.ingredientCount) && Objects.equals(ingredients, dish.ingredients);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(dishId, dishName, caloriesCount, ingredientCount, ingredients);
+    }
+
     @Override
     public String toString() {
-        return "Dish{" +
-                "dishName='" + dishName + '\'' +
+        return "Dish:" +
+                "dishId=" + dishId +
+                ", dishName='" + dishName + '\'' +
                 ", caloriesCount=" + caloriesCount +
                 ", ingredientCount=" + ingredientCount +
-                ", ingredients=" + ingredients +
-                '}';
+                ", ingredients=" + ingredients;
```

Comment:
How it will look like as a string? This way of output will not work properly for Objects

Reply:
I will replace it with this code:
ingredients.stream().map(Objects::toString).collect(Collectors.toList());

Because ingredients is a List.

Reply:
I didn't get how it can improve a situation. You still returning an object and trying to print it.